x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx

```diff
@@ -141,17 +149,33 @@ export const getRemediationList = (rule: CspFinding['rule']) => [
 
 const getEvidenceList = ({ result }: CspFinding) =>
   [
-    result.expected && {
-      title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.expectedTitle', {
-        defaultMessage: 'Expected',
-      }),
-      description: <CodeBlock>{JSON.stringify(result.expected, null, 2)}</CodeBlock>,
-    },
     {
-      title: i18n.translate('xpack.csp.findings.findingsFlyout.overviewTab.actualTitle', {
-        defaultMessage: 'Actual',
-      }),
-      description: <CodeBlock>{JSON.stringify(result.evidence, null, 2)}</CodeBlock>,
+      description: (
+        <>
+          <EuiText color="subdued">
+            <FormattedMessage
+              id="xpack.csp.findings.findingsFlyout.overviewTab.evidenceDescription"
+              defaultMessage="The specific resource metadata that was evaluated to generate this posture finding"
+            />
+          </EuiText>
+          <EuiSpacer size={'s'} />
+          <div style={{ height: 400 }}>
+            <CodeEditor
+              isCopyable
+              allowFullScreen
+              languageId={XJsonLang.ID}
+              value={JSON.stringify(result.evidence, null, 2)}
+              options={{
+                readOnly: true,
+                lineNumbers: 'on',
+                folding: true,
+                automaticLayout: true,
+              }}
+            />
+            {/* <CodeBlock>{JSON.stringify(result.evidence, null, 2)}</CodeBlock>*/}
```

Comment:
@JordanSh  do you need this comment below?

Reply:
nope, thats just until @tinnytintin10 will decide which version he prefers. will either remove this or the code editor before merge.